lib/election.js

```diff
@@ -417,32 +408,48 @@ function _createEventManifest(ledgerNode, blockHeight, callback) {
  *          completes.
  */
 function _recommendElectors(ledgerNode, voter, electors, manifest, callback) {
-  // TODO: use given parameters and blockchain to determine a set of
-  // electors to recommend for the next block
```

Comment:
Let's keep this TODO around for later.

Reply:
I'll put it back in, but what does that comment mean?  Seems like a description of what the function is doing already to me.

Reply:
Yeah, it does describe what the function is already doing, so we should be more specific:

```js
// TODO: determine if a better deterministic algorithm (better == more secure, performant)
// could be used here that uses given parameters and blockchain data to recommend
// electors for the next block
```

Reply:
done.